File: Python/programmers/lv1/시저 암호.py
def solution(s, n):
    answer = ''
    for c in s:
        if c == ' ':
            answer += ' '
        elif c.isupper():
            answer += chr((ord(c) - ord('A') + n) % 26 + ord('A'))
        else:
            answer += chr((ord(c) - ord('a') + n) % 26 + ord('a'))
    return answer

File: kotlin_java/src/programmers/lv1/시저_암호.java
package programmers.lv1;

public class 시저_암호 {
    public String solution(String s, int n) {
        String answer = "";

        for(int i = 0; i<s.length(); i++)
        {
            if(s.charAt(i) == ' ')
                answer = answer + " ";
            else
            {
                if(s.charAt(i) < 'a' && s.charAt(i) + n > 'Z')
                    answer = answer + (char)(s.charAt(i)+n+'A'-'Z'-1);
                else if(s.charAt(i) + n > 'z')
                    answer = answer + (char)(s.charAt(i)+n+'a'-'z'-1);
                else
                    answer = answer + (char)(s.charAt(i)+n);
            }
        }
        return answer;
    }
}
